diff --git a/node_modules/expo-widgets/ios/Widgets/WidgetLiveActivity.swift b/node_modules/expo-widgets/ios/Widgets/WidgetLiveActivity.swift
index 9eae718..40cc151 100644
--- a/node_modules/expo-widgets/ios/Widgets/WidgetLiveActivity.swift
+++ b/node_modules/expo-widgets/ios/Widgets/WidgetLiveActivity.swift
@@ -3,10 +3,10 @@ import WidgetKit
 import ExpoModulesCore
 import ActivityKit
 
-struct LiveActivityAttributes: ActivityAttributes {
+public struct LiveActivityAttributes: ActivityAttributes {
   public struct ContentState: Codable, Hashable {
-    var name: String
-    var props: String
+    public var name: String
+    public var props: String
   }
 }
 
